tools/compile_tool/main.cpp

```diff
@@ -202,10 +202,14 @@ static std::map<std::string, std::string> parseConfigFile(char comment = '#') {
 
 static std::map<std::string, std::string> configure() {
     const bool isMYRIAD = FLAGS_d.find("MYRIAD") != std::string::npos;
-
     auto config = parseConfigFile();
 
     if (isMYRIAD) {
+        if (config[InferenceEngine::MYRIAD_DISABLE_MX_BOOT] == "YES") {
+            // the second "YES" is used to make sure we are working with compile_tool
+            // and not affecting other applications.
+            config[InferenceEngine::MYRIAD_DISABLE_MX_BOOT] = "YES,YES";
```

Comment:
I think it is a redundant action. Please, revert logic to keep only 1 "YES".
It is a private plugin option and a user should understand what he or she does.

Reply:
Okay, reverted